integration-tests/src/tests/defaults.rs

```diff
@@ -54,11 +54,13 @@ pub fn ipfs_hash() -> BoundedVec<u8, ConstU32<96>> {
 pub fn default_weights() -> Vec<u8> {
 	vec![20u8, 15u8, 10u8, 25u8, 30u8]
 }
-pub fn default_bidder_multipliers() -> Vec<u8> {
-	vec![1u8, 6u8, 10u8, 8u8, 3u8]
+pub fn default_bidder_modes() -> Vec<ParticipationMode> {
+	use ParticipationMode::{Classic, OTM};
```

Comment:
Let's move this `use` outside, in this way we should be able to avoid this double `use`.

EDIT: I see that this `use` is used plenty of times across several files, let's try to import it at the highest level possible to minimize the number of imports required